toolchest_client/files/tests/test_s3.py

```diff
@@ -5,12 +5,14 @@
 
 
 def test_s3_params():
-    example_s3_uri = "s3://toolchest-public-examples/example.fastq"
+    example_s3_uri = "s3://toolchest-public-examples/dummy-id/example.fastq"
     params = get_params_from_s3_uri(example_s3_uri)
     target_params = {
-        "arn": "arn:aws:s3:::toolchest-public-examples/example.fastq",
+        "arn": "arn:aws:s3:::toolchest-public-examples/dummy-id/example.fastq",
```

Comment:
Worth noting that we're now expecting the uuid to be present in the S3 URI structure. This isn't reflected in the db schema rn, where the uuid is just a part of the file prefix.